scripts/filter_expected_warnings.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+"""Script to filter out old warnings from doxygen."""
+
+import sys
+import re
+
+
+def compare_lists(list1, list2):
```

Comment:
Why not `list1 == list2`?

Reply:
Good idea, thanks.  (Google didn't tell me I could do that!)